docs/tutorials/advanced/geonode_production/geosites.txt

```diff
@@ -84,6 +83,32 @@ Permissions by Site
 ===================
 By default GeoNode is publicly available. In the case of GeoSites, new data will be publicly available, but only for the site it was added to, and the master site (all data is added to the master site). 
 
+Activate geosites app
+================
+In order to use geonode in Multi-tenancy mode, follow these steps:
+1. check in settings.py if 'geonode.contrib.geosites' in GEONODE_CONTRIB_APPS is rightly uncommented
+2. add in settings.py 'geonode.contrib.geosites' in INSTALLED_APPS
+3. run python manage.py syncdb
+
 Adding New Sites
 ================
-A management command exists to easily create a new site.  This will create all the needed directories, as well as a site specific settings file.  The command may also create a website configuration file.
+To add a new site follow the following steps:
```

Comment:
Thanks Eugenio, this PR is good so far. But the following configuration seems to reflect your setup.
The ideal setup is to have geosites outside the geonode code by using a geonode-project for example. It is quite important also to have the statics elsewhere like in /var/www subfolders as apache will serve them